src/features/flows/nRF9151/index.ts

```diff
@@ -287,6 +287,15 @@ const learnConfig = [
             href: 'https://docs.nordicsemi.com/bundle/nwp_044/page/WP/nwp_044/intro.html',
```

Comment:
Please replace the Best practices with the NCS-specific section suggested in #244 .